pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -375,25 +376,51 @@ func GetBootstrapOpenshiftRoleBindings(openshiftNamespace string) []authorizatio
 	}
 }
 
-func GetBootstrapClusterRoleBindings() []authorizationapi.ClusterRoleBinding {
-	return []authorizationapi.ClusterRoleBinding{
+func GetBootstrapServiceAccountRoleBindings(namespace string) []authorizationapi.RoleBinding {
```

Comment:
This file is all about objects that are created once per cluster, not once per project.  The package is fine, but different file perhaps and something to indicate that these should be created for each project.